notebooks/new-calibrationStarVisit.ipynb

```diff
@@ -18,6 +18,19 @@
     "import lsst.observing.commands.calibrationStarVisit as calibrationStarVisit"
    ]
   },
+  {
```

Comment:
Can you clear all the outputs, please?

Reply:
I'm a little confused here as I thought the outputs were all clear... I'll re-clear them